### PR TITLE
avalanche's k8s deployment yaml file

### PR DESCRIPTION
**Pull request by ravishankarsrrav**

### CODE REVIEW

File: kubernetes-deployment-yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+metadata:
+  name: pg-avalanche
+  namespace: avalanche
+  labels:
+    name: avalanche
+spec:
+  selector:
+    matchLabels:
+      app: pg-avalanche
+  replicas: 1 # tells deployment to run 2 pods matching the template
```
Comment (csmarchbanks) [inline kubernetes-deployment-yaml]: The comment here is misleading.

File: kubernetes-deployment-yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  template:
+    metadata:
+      labels:
+        app: pg-avalanche
+    spec:
+      containers:
+      - name: pg-avalanche
+        image: quay.io/freshtracks.io/avalanche
```
Comment (csmarchbanks) [inline kubernetes-deployment-yaml]: Perhaps add a `:latest` tag to always pull down the latest image?

File: kubernetes-deployment-yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+kind: StatefulSet
```
Comment (csmarchbanks) [inline kubernetes-deployment-yaml]: Any reason this is a StatefulSet instead of a Deployment? If not I find Deployments easier to work with.

**Comment by ravishankarsrrav** [comment]
Thank you for reviewing the contribution. I have modified the yaml file as changes suggested by you. It makes sense to keep it in a separate folder so I have created a new k8s folder and added the deployment yaml file.

**Comment by csmarchbanks** [comment]
This looks great, thanks for your contribution!

**Pull request merged**